_includes/footer.html

```diff
@@ -17,7 +17,7 @@
             <a href="https://twitter.com/getbootstrap" class="twitter-follow-button" data-link-color="#0069D6" data-show-count="true">Follow @getbootstrap</a>
           </li>
           <li>
-            <a href="https://twitter.com/share" class="twitter-share-button" data-url="http://twbs.github.com/bootstrap/" data-count="horizontal" data-via="getbootstrap" data-related="mdo:Creator of Twitter Bootstrap">Tweet</a>
+            <a href="https://twitter.com/share" class="twitter-share-button" data-url="https://twbs.github.com/bootstrap/" data-count="horizontal" data-via="getbootstrap" data-related="mdo:Creator of Twitter Bootstrap">Tweet</a>
```

Comment:
This should just be getbootstrap.com
We don't reference twbs.github.{io,com} anywhere directly anymore.